### PR TITLE
add missing copyright to various files

### PR DESCRIPTION
**Pull request by andife**
### Description

add missing copyright to various files

used command:
reuse annotate --no-replace --copyright="Copyright (c) ONNX Project Contributors" 

#### Motivation and Context
Purpose: Make onnx https://reuse.software/ compliant

### CODE REVIEW

**Review by gramalingam — APPROVED** [review]

**Review by jcwchen — APPROVED** [review]

**Comment by justinchuby** [comment]
Do we need to create a lintrunner lint to ensure new files have this? Something similar to https://github.com/onnx/onnx/blob/main/.lintrunner.toml#L164-L184

File: onnx/reference/ops/__init__.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# Copyright (c) ONNX Project Contributors
+
```
Comment (justinchuby) [inline onnx/reference/ops/__init__.py:2]: Redundant?

Comment (andife) [reply onnx/reference/ops/__init__.py:2]: yes you are right. I will change that.

**Comment by andife** [comment]
> Do we need to create a lintrunner lint to ensure new files have this? Something similar to https://github.com/onnx/onnx/blob/main/.lintrunner.toml#L164-L184

Yes something like that was my intention, but first I would like to have the repo adapted or cleared if necessary. Then  would like to use a tool that was developed for that purpose. 

Then there is the question whether fossology in a CI (I think that is still too complicated from my point of view), so I would like to use the Github plugin from https://github.com/marketplace/actions/reuse-compliance-check (but I would discuss that first).

**Pull request merged**